components/campus-news-card.vue

```diff
@@ -37,11 +37,13 @@ export default {
   computed: {
     ...mapState({
       campusNews: (state) => state.news.campusNews,
-      browserStateReady: (state) => state.browserStateReady,
     }),
   },
   mounted() {
-    this.loadCampusNews();
+    if (this.lazyLoad) {
```

Comment:
Im moment wird im statischen build nichts geladen, weil "this.lazyload" unbekannt ist"

Reply:
oops